package.json

```diff
@@ -21,6 +24,7 @@
   "homepage": "https://github.com/braxnu/vineverse#readme",
   "dependencies": {
     "express": "^4.17.1",
+    "npm-watch": "^0.6.0",
```

Comment:
dziala bez tego, do watchowania serwera uzywamy nodemona, a do klienta uzywamy webpack w trybie --watch, a nasze skrypty watch to tylko nazwy nadane przez nas, moglyby sie nazywac np, obserwuj:klient

Reply:
Poprawione